docs/administrator-documentation/references/repos-csv.md

```diff
@@ -0,0 +1,119 @@
+| java        | `false`  | Configures the JDK to use.                                                                                                                                                                                                                             | `17` or `17-tem` or `17.0.6-tem`                                              |
+| jvmopts     | `false`  | JVM options added to tools building LSTs. Must be configured before you can run the build command if non-standard VM options are required.                                                                                                             | `-Xmx4G`                                                                      |
+| mavenArgs   | `false`  | Build arguments are added to the end of the Maven command line when building LSTs.                                                                                                                                                                     | `-Pfast`                                                                     |
+| gradleArgs  | `false`  | Build arguments that are added to the end of the Gradle command line when building LSTs.                                                                                                                                                               | `-Dmyprop=myvalue`                                                            |
+| org*        | `false`  | If you want to configure an organizational hierarchy, you can provide one or more organization columns. Each column will specify an organization the repository should be part of. The column name should be `org` plus a number such as: `org1,org2,org3`.  | `openrewrite`                                                           |
+</details>
+
+## Generating this file
+
+To kick-start a `repos.csv` file, we recommend using "[repo fetchers](https://github.com/moderneinc/repository-fetchers)". Repo fetchers are scripts that generate a CSV file of all of your repositories by making API calls to your source code manager.
+
+## Defining an organizational hierarchy
+
+Org hierarchy is best explained with an example. Consider an organization that consists of two teams and two directors:
+
+```
+All
+├── Director A
+|       └── Team 1
+|       └── Team 2
+├── Director B
+
+```
+
+The following CSV file would represent this organizational structure:
+
+```bash showLineNumbers
+cloneUrl,branch,org1,org2,org3
+"https://github.com/apache/maven-doxia","master","Team 1","Director A","ALL"
+"https://github.com/Netflix/photon","main","Team 2","Director A","ALL"
+"https://github.com/Netflix/ribbon","master","Director A","ALL"
+"https://github.com/apache/maven-doxia","master","Director B","ALL"
```

Comment:
```suggestion
"https://github.com/apache/maven-doxia","main","Team 1","Director A","ALL"
"https://github.com/Netflix/photon","master","Team 2","Director A","ALL"
"https://github.com/Netflix/ribbon","main","Director A","ALL"
"https://github.com/apache/maven-doxia","main","Director B","ALL"
```

As `main` is no more common than `master`, I would inverse the use of the branches here.

Reply:
While I appreciate the thought, I am using the branches that are used in these repos. I don't think we need to worry about demonstrating main/master for people. Also, most old code uses master instead of main as people often will not change the branch.